core/modules/filters/is.js

```diff
@@ -27,40 +27,44 @@ Export our filter function
 */
 exports.is = function(source,operator,options) {
 
+	var isFilterOperators = getIsFilterOperators(),
+		subops = operator.operand.split("+");
 
-	if( !operator.operand) {
-		// Return all tiddlers if the operand is missing
+	//Make sure all the operands are defined.
+	for (var t = 0; t < subops.length; t++){
```

Comment:
subops.length is used 4 times. So we may copy it to it's own variable, for a minimal speed boost.